src/features/user/userSlice.js

```diff
@@ -85,10 +88,28 @@ export const userSlice = createSlice({
     startUploading: state => {
       state.uploadingPhoto = true;
     },
+    searchUser: (state, { payload }) => {
+      state.searchTerm = payload;
+      state.foundUsers = state.users.filter(
+        user =>
+          user.username.toLowerCase().includes(payload.trim().toLowerCase()) ||
+          user.firstName.toLowerCase().includes(payload.trim().toLowerCase()) ||
+          user.lastName.toLowerCase().includes(payload.trim().toLowerCase())
+      );
```

Comment:
Search functionality is amazing. Works smooth.

Reply:
thanks